src/EditorFeatures/VisualBasic/EndConstructGeneration/EndConstructStatementVisitor_Properties.vb

```diff
@@ -122,6 +122,10 @@ Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.EndConstructGeneration
         ''' might be in the middle of typing one that would be a false positive. </param>
         Private Function NeedsGetAccessor(propertyDeclaration As PropertyStatementSyntax, Optional accessorToIgnore As AccessorBlockSyntax = Nothing) As Boolean
             Dim propertyBlock = propertyDeclaration.GetAncestor(Of PropertyBlockSyntax)()
+            If propertyDeclaration.Modifiers.Any(Function(m) m.IsKind(SyntaxKind.WriteOnlyKeyword)) Then
```

Comment:
I would remove the code on line 139, and move this check before the declaration of `propertyBlock`